packages/grid/data-grid/src/tests/filtering.DataGrid.test.tsx

```diff
@@ -189,12 +189,51 @@ describe('<DataGrid /> - Filter', () => {
 
   describe('date operators', () => {
     [
-      { operator: 'is', value: new Date(2000, 11, 1), expected: ['12/1/2000'] },
-      { operator: 'not', value: new Date(2000, 11, 1), expected: ['1/1/2001', '1/1/2002'] },
-      { operator: 'after', value: new Date(2001, 0, 1), expected: ['1/1/2002'] },
-      { operator: 'onOrAfter', value: new Date(2001, 0, 1), expected: ['1/1/2001', '1/1/2002'] },
-      { operator: 'before', value: new Date(2001, 0, 1), expected: ['12/1/2000'] },
-      { operator: 'onOrBefore', value: new Date(2001, 0, 1), expected: ['12/1/2000', '1/1/2001'] },
+      { operator: 'is', value: '' },
+      { operator: 'is', value: undefined },
+      { operator: '', value: '2000-12-01' },
+      { operator: '', value: '' },
```

Comment:
Is it possible for the `operatorValue` to be empty? In the UI, it doesn't seem possible.

Reply:
Using the `filterModel` prop it can be empty.

Reply:
This sounds indeed like a good possible opportunity for follow-ups.